salt/metalk8s/salt/master/files/master-99-metalk8s.conf.j2

```diff
@@ -25,7 +25,9 @@ no_host_keys: true
 rest_cherrypy:
   port: 4507
   host: {{ salt_ip }}
-  disable_ssl: true
+  ssl_crt: /etc/kubernetes/pki/salt-api.crt
```

Comment:
Should not we put those paths into some pillars or something to make them defined at one location only?